### PR TITLE
Required changes after adding annotations

### PR DESCRIPTION
**Pull request by artek-koltun**
See https://github.com/opiproject/opi-api/pull/321

### CODE REVIEW

**Comment by artek-koltun** [comment]
CI waits for godpu update https://github.com/opiproject/godpu/pull/264

File: pkg/frontend/nvme_subsystem_test.go [hunk 1]
```diff
@@ -309,9 +309,7 @@ func TestFrontEnd_UpdateNvmeSubsystem(t *testing.T) {
 	}{
 		"invalid fieldmask": {
 			&fieldmaskpb.FieldMask{Paths: []string{"*", "author"}},
-			&pb.NvmeSubsystem{
-				Name: testSubsystemName,
```
Comment (glimchb) [inline pkg/frontend/nvme_subsystem_test.go:313]: not sure this is correct... in update we need Name... which is missing from `testSubsystem`

File: pkg/frontend/nvme_subsystem_test.go [hunk 2]
```diff
@@ -320,9 +318,7 @@ func TestFrontEnd_UpdateNvmeSubsystem(t *testing.T) {
 		},
 		"unimplemented method": {
 			nil,
-			&pb.NvmeSubsystem{
-				Name: testSubsystemName,
```
Comment (glimchb) [inline pkg/frontend/nvme_subsystem_test.go:324]: same as above

File: pkg/kvm/blk_test.go [hunk 3]
```diff
@@ -15,14 +15,19 @@ import (
...
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
 var (
 	testVirtioBlkID            = "virtio-blk-42"
 	testVirtioBlkName          = server.ResourceIDToVolumeName(testVirtioBlkID)
 	testCreateVirtioBlkRequest = &pb.CreateVirtioBlkRequest{VirtioBlkId: testVirtioBlkID, VirtioBlk: &pb.VirtioBlk{
-		Name:          "",
```
Comment (glimchb) [inline pkg/kvm/blk_test.go:24]: empty name can be removed

**Review by glimchb — APPROVED** [review]
1 comment, otherwise looks good
please update godpu hash, so ci can pass
https://github.com/opiproject/godpu/pkgs/container/godpu

File: pkg/frontend/nvme_subsystem_test.go (hunk 1 above)
Comment (artek-koltun) [reply pkg/frontend/nvme_subsystem_test.go:313]: Added as a comment to this issue:
https://github.com/opiproject/opi-spdk-bridge/issues/443

**Pull request merged**